tmt/export.py

```diff
@@ -0,0 +1,189 @@
+
+        # Default tester
+        try:
+            email_address = email.utils.parseaddr(fmf_case_attrs['contact'])[1]
+            # TODO handle nitrate user not existing and other stuff
+            nitrate_case.tester = nitrate.User(email_address)
+            echo(style('default tester: ', fg='green') + email_address)
+        except KeyError:
+            # Defaults to author of the test case
+            pass
+
+        # Duration
+        try:
+            nitrate_case.time = fmf_case_attrs['duration']
+            echo(style('estimated time: ', fg='green') +
+                 fmf_case_attrs['duration'])
+        except KeyError:
+            # Defaults to 5 minutes
+            nitrate_case.time = '5m'
+            echo(style('estimated time: ', fg='green') + '5m')
+
+        # Status
+        try:
+            if fmf_case_attrs['enabled']:
+                nitrate_case.status = nitrate.CaseStatus('CONFIRMED')
+                echo(style('status: ', fg='green') + 'CONFIRMED')
+            else:
+                nitrate_case.status = nitrate.CaseStatus('DISABLED')
+                echo(style('status: ', fg='green') + 'DISABLED')
+        except KeyError:
+            # Defaults to enabled
+            nitrate_case.status = nitrate.CaseStatus('CONFIRMED')
+            echo(style('status: ', fg='green') + 'CONFIRMED')
+
+        # Environment
+        try:
+            env = ' '.join(tmt.utils.dict_to_shell(
+                fmf_case_attrs['environment']))
+            nitrate_case.arguments = env
+            echo(style('arguments: ', fg='green') + env)
+        except KeyError:
+            # FIXME unable to set empty arguments, possibly error in xmlrpc, BZ#1805687
+            nitrate_case.arguments = ' '
+            echo(style('arguments: ', fg='green') + "' '")
+
+        struct_field = tmt.utils.StructuredField(nitrate_case.notes)
+        echo(style('Structured Field: ', fg='blue'))
+
+        # Mapping of structured field sections to fmf case attributes
+        section_to_attr = {'relevancy': 'relevancy', 'description': 'summary',
+                           'purpose-file': 'description', 'hardware': 'extra-hardware', 'pepa': 'extra-pepa'}
+
+        for section, attribute in section_to_attr.items():
+            try:
+                struct_field.set(section, fmf_case_attrs[attribute])
+                echo(style(section + ': ', fg='green') +
+                     fmf_case_attrs[attribute].strip())
+            except KeyError:
+                pass
+
+        # fmf identifer
+        fmf_id = create_fmf_id(path=os.path.abspath('.'),
+                               name=fmf_case.name)[0]
+
+        struct_field.set('fmf', pprint.pformat(fmf_id))
+        echo(style('fmf: ', fg='green') + pprint.pformat(fmf_id))
+
+        fmf_warning = """Test case was ported to fmf and is maintaned in git.
+Any changes made here might be overwritten.
+More information here: https://tmt.readthedocs.io/en/latest/stories/cli.html#id14\n\n"""
```

Comment:
Not sure about the link. It's long and leads to an anchor which can possibly change. Either add a new section under [questions](https://tmt.readthedocs.io/en/latest/questions.html) or remove it? Or add a named anchor? I don't think we should reference stories here.

Reply:
What else do you think should be there?
Link with some further explanation is in my opinion essential (information about case being ported is already in Structured Field)

Reply:
For now let's add a new section to the [questions](https://github.com/psss/tmt/blob/master/docs/questions.rst) page as suggested above.

Reply:
And should the fmf warning contain link to the new question? Without link to further explanation the warning seems pointless to me.

Reply:
Yes, the link should be included in the test case notes.

Reply:
Improved by c635729
Please check it if it's better now.

Reply:
Thanks! Looks much better now. Let's just use more concise heading so that we have a shorter link. What about `Nitrate Migration`. I think that would be just enough.